proxy/docker/build-and-push.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright 2018-2022 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+#
+# Usage: $ ./build-and-push.sh <version>
+
+set -eu
+
+readonly SEMVER_REGEX="^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?$" # X.Y.Z
+readonly ORG="openlineage"
+
+# Change working directory to proxy module
+project_root=$(git rev-parse --show-toplevel)
+cd "${project_root}/proxy"
```

Comment:
since this script is specific to the proxy, I'd either a) rename the script to `build-and-push-proxy.sh` or b) make the script generic to support arguments for the directory and the image name

Reply:
I think for now, I'll use `build-and-push-proxy.sh` until the need to make the script more generic.